src/foundry/client/data/documents/region.d.mts

```diff
@@ -21,25 +21,74 @@ declare global {
     interface RegionEvent {
       /** The name of the event */
       name: string;
+
       /** The data of the event */
       data: object;
+
       /** The Region the event was triggered on */
       region: RegionDocument;
+
       /** The User that triggered the event */
-      user: User
+      user: User;
     }
 
     interface SocketRegionEvent {
       /** The UUID of the Region the event was triggered on */
       regionUuid: string;
+
       /** The ID of the User that triggered the event */
       userId: string;
+
       /** The name of the event */
       eventName: string;
+
       /** The data of the event */
       eventData: object;
+
       /** The keys of the event data that are Documents */
-      eventDataUuids: string[]
+      eventDataUuids: string[];
+    }
+
+    interface _UpdateTokenOptions {
```

Comment:
You can remove the `_`. I know the method is named `_updateToken` but I personally think it's more confusing with an `_`.